src/foremast/elb/create_elb.py

```diff
@@ -82,6 +82,11 @@ def make_elb_json(self):
         security_groups.append(self.app)
         security_groups.extend(self.properties['security_group']['elb_extras'])
 
+        for each_sg, duplicate_sg_name in SECURITYGROUP_REPLACEMENTS.items():
```

Comment:
I think we need a test case to demonstrate what is supposed to happen. Are we:
* Replacing a Security Group with another Security Group of a different name?
* Removing `key` and adding `value`?
* Removing `value` and adding `key`?

Possible outcomes to clarify:
```python
security_groups = ['a', 'b', 'c', 'd']

replacements = {'a': 'd'}
for key_sg, value_sg in replacements.items():
    …

# Which assertions are correct?
assert security_groups == ['b', 'c']
assert security_groups == ['b', 'c', 'd']
assert security_groups == ['a', 'b', 'c']
assert security_groups == ['a', 'b', 'c', 'd']
```

Reply:
Done. Moved to utils.

Writing a test case I suppose around this.